site/README.md

```diff
@@ -1,18 +0,0 @@
-# IPFS Distributions
-
-
-## Development
-
-Harpjs setup with
-
-* Live reload using browser-sync
-* JavaScript
-  * bundeling using webpack
-  * transpiling using babel
-* Linting using eslint
-
-
-```bash
-$ gulp
-$ open localhost:3000
-```
```

Comment:
I strongly disagree that make files are "nice" they are hard to understand for a lot of people, even I struggle with them everytime I look at them. In addition they require a lot of overhead and understanding if you ever try runnin on something like windows. 

But I don't want to make this pr about a discussion about make vs other build systems. As stated before this is a work in progress and at the end  things will just work by running `make build` as that was what was setup initially. But that will do things like `npm install` as there javascript dependencies in this repo that need to be installed before you can do anything. 

Also one question who is this "user" you are talking about, do you mean yourself and @whyrusleeping (because those are the main people actually using this besides myself right now). Or are you talking about future contributors? In that case see my argument against `make` as that constrains all contributors to be running on unix platforms or do a lot of setup work and we should think about moving everything to be node based instead as well as dropping shell scripts with shebangs which don't work on windows by default either.